src/config/dark-sites.config

```diff
@@ -319,6 +319,7 @@ dicebear.com
 diep.io
 digi77.com
 digikam.org
+dignityofwar.com
```

Comment:
This part is definitely not DARK.
![obraz](https://github.com/user-attachments/assets/b16c083c-8ffe-4165-9861-f9e04bb9003e)

I can accept only wiki.

Reply:
Whoops, you're entirely right sorry!